src/TextButton.cc

```
@@ -49,10 +49,18 @@ void TextButton::paintIcon(QPainter *painter, const QRectF &iconRect, const qrea
     painter->setPen(foregroundColor());
     painter->setRenderHint(QPainter::TextAntialiasing);
 
+    // Compute stable, physical-pixel-aligned dimensions to prevent subpixel text blurring
+    const QPointF p0 = snapper.snap(QPointF(0.0, 0.0));
```

Clang (14.0.6)

[warning] variable 'p0' is not initialized

(cppcoreguidelines-init-variables)